harvester/gene_enricher.py

```diff
@@ -15,22 +15,40 @@
         'ensembl_gene_id': doc.get('ensembl_gene_id', None),
         'entrez_id': doc.get('entrez_id', None)
         }
-    GENES[doc['symbol']] = gene
+    GENES[doc['symbol']] = [gene]
     if gene['ensembl_gene_id']:
-        ALIASES[gene['ensembl_gene_id']] = gene
+        if gene['ensembl_gene_id'] not in ALIASES:
+            ALIASES[gene['ensembl_gene_id']] = []
+        ALIASES[gene['ensembl_gene_id']].append(gene)
     if gene['entrez_id']:
-        ALIASES[gene['entrez_id']] = gene
+        if gene['entrez_id'] not in ALIASES:
+            ALIASES[gene['entrez_id']] = []
+        ALIASES[gene['entrez_id']].append(gene)
     for alias in doc.get('alias_symbol', []):
-        ALIASES[alias] = gene
+        if alias not in ALIASES:
+            ALIASES[alias] = []
+        ALIASES[alias].append(gene)
     for prev in doc.get('prev_symbol', []):
-        ALIASES[prev] = gene
+        if prev not in ALIASES:
+            ALIASES[prev] = []
+        ALIASES[prev].append(gene)
 data = None
 
 
-def get_gene(identifier):
+def get_genes(identifier):
     """ return gene for identifier """
     for store in [GENES, ALIASES]:
-        gene = store.get(identifier, None)
-        if gene:
-            return gene
+        genes = store.get(identifier, None)
```

Comment:
Perhaps we should use `[]` as the default value here, it doesn't affect line 42, and has the added property of allowing for length checks.